project_task_dependency/views/project_task_view.xml

```diff
@@ -13,10 +13,9 @@
                     <field name="recursive_dependency_task_ids" invisible="1"/>
                     <field name="dependency_task_ids"
                            domain="[
-                                   ('id', '!=', id),
-                                   ('id', 'not in', recursive_dependency_task_ids and recursive_dependency_task_ids[0][2] or [])
-                                   ]"
-                    >
+                                ('id', '!=', id),
```

Comment:
Better to put the domain in only one line for avoiding strange indentation on backend when editing the view

Reply:
Done